dot/sync/tip_syncer_integeration_test.go

```diff
@@ -261,12 +270,20 @@ func TestTipSyncer_handleTick_case3(t *testing.T) {
 			targetNumber: uintPtr(fin.Number),
 			direction:    network.Descending,
 			requestData:  bootstrapRequestData,
-			pendingBlock: s.pendingBlocks.getBlock(header.Hash()),
+			pendingBlock: &pendingBlock{
+				hash:    header.Hash(),
+				number:  300,
+				header:  header,
+				body:    &types.Body{},
+				clearAt: time.Time{},
+			},
 		},
 	}
 
 	w, err = s.handleTick()
 	require.NoError(t, err)
+	assert.Greater(t, w[0].pendingBlock.clearAt, time.Now())
```

Comment:
nit just to be safe and avoid panics

```suggestion
	require.NotEmpty(t, w)
	assert.Greater(t, w[0].pendingBlock.clearAt, time.Now())
```